### PR TITLE
Allow disabling authentication

### PR DESCRIPTION
**Pull request by roedie**
This will allow running a solr server withouth authentication

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,6 +49,8 @@ Requirements
     default: `{{ dest_main_path }}\\solr-{{ solr_version }}`
   - `change_default_password` - to change default password (will be solr/SolrRocks)
     default: `True`
+  - `solr_auth_configure` - Enable authentication
+    default: `true`
```
Comment (ViachaslauKabak) [inline README.md]: please correct to True

**Comment by ViachaslauKabak** [comment]
Please resolve merge conflicts

**Comment by roedie** [comment]
Yes I noticed it. I will. Have to do some other stuff first ;-)

**Comment by roedie** [comment]
Done

**Pull request merged**